pkg/model/stack.go

```diff
@@ -36,14 +34,14 @@ func (s Stack) FindOptionalParameter(name string) (StackOptionalParameter, error
 }
 
 type StackRequiredParameter struct {
-	Name      string `gorm:"primaryKey"`
-	StackName string `gorm:"primaryKey"`
-	Consumed  bool
+	Name      string `gorm:"primaryKey" json:"name"`
```

Comment:
These are done automatically by the `json.Marshaler` so no json tag would be needed. We can stick to being consistent and still add them like you did though 👌🏻 😄